presto-memory-context/src/test/java/com/facebook/presto/memory/context/TestMemoryContexts.java

```diff
@@ -162,14 +165,15 @@ public void testClosedLocalMemoryContext()
     public void testClosedAggregateMemoryContext()
     {
         AggregatedMemoryContext aggregateContext = newSimpleAggregatedMemoryContext();
-        LocalMemoryContext localContext = aggregateContext.newLocalMemoryContext();
+        LocalMemoryContext localContext = aggregateContext.newLocalMemoryContext("test");
         aggregateContext.close();
         localContext.setBytes(100);
     }
 
     private static class TestMemoryReservationHandler
             implements MemoryReservationHandler
     {
+        private final Map<String, Long> taggedMemoryAllocations = new HashMap<>();
```

Comment:
It appears like this field is never used. Remove the "updateTaggedAllocations" here?